env/Lib/site-packages/PIL/ImageQt.py

```
@@ -0,0 +1,219 @@
+    exclusive_fp = False
+
+    # handle filename, if given instead of image name
+    if hasattr(im, "toUtf8"):
+        # FIXME - is this really the best way to do this?
+        im = str(im.toUtf8(), "utf-8")
+    if is_path(im):
+        im = Image.open(im)
+        exclusive_fp = True
+    assert isinstance(im, Image.Image)
+
+    qt_format = getattr(QImage, "Format") if qt_version == "6" else QImage
```

Ruff (0.8.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)